vendor/bat-native-ledger/src/bat/ledger/internal/credentials/credentials_sku.cc

```diff
@@ -291,7 +271,7 @@ void CredentialsSKU::ClaimStatusSaved(
     const CredentialsTrigger& trigger,
     ledger::ResultCallback callback) {
   if (result != ledger::Result::LEDGER_OK) {
-    BLOG(0, "Claim status not saved");
+    BLOG(0, "Claim status not saved " << result);
```

Comment:
Nit: Maybe a colon here to separate the result from the message?

Reply:
fixed